tests/orc_dataloader/orc_dataloader.t

```diff
@@ -0,0 +1,107 @@
+// orc_dataloader.t
+
+-1"\n+----------|| Import the arrowkdb library ||----------+\n";
+\l q/arrowkdb.q
+
+-1"\n+----------|| Filesystem functions for Linux/MacOS/Windows ||----------+\n";
+rm:{[filename] $[.z.o like "w*";system "del ",filename;system "rm ",filename]};
+
+-1"\n+----------|| Create the datatype identifiers ||----------+\n";
+ts_dt:.arrowkdb.dt.timestamp[`nano];
+
+i8_dt:.arrowkdb.dt.int8[];
+i16_dt:.arrowkdb.dt.int16[];
+i32_dt:.arrowkdb.dt.int32[];
+i64_dt:.arrowkdb.dt.int64[];
+
+-1"\n+----------|| Create the field identifiers ||----------+\n";
+ts_fd:.arrowkdb.fd.field[`tstamp;ts_dt];
+
+i8_fd:.arrowkdb.fd.field[`int8;i8_dt];
+i16_fd:.arrowkdb.fd.field[`int16;i16_dt];
+i32_fd:.arrowkdb.fd.field[`int32;i32_dt];
+i64_fd:.arrowkdb.fd.field[`int64;i64_dt];
+
+-1"\n+----------|| Create the schemas for the list of fields ||----------+\n";
+dataloader_schema:.arrowkdb.sc.schema[(ts_fd,i8_fd,i16_fd,i32_fd,i64_fd)];
+
+-1"\n+----------|| Number of items in each array ||----------+\n";
+N:10
+
+-1"\n+----------|| Create data for each column in the table ||----------+\n";
+ts_data:asc N?0p;
+
+i8_data:N?0x64;
+i16_data:N?100h;
+i32_data:N?100i;
+i64_data:N?100;
+
+-1"\n+----------|| Combine the data for all columns ||----------+\n";
+dataloader_data:(ts_data;i8_data;i16_data;i32_data;i64_data);
+
+-1"\n+----------|| Write the schema and array data to a parquet file ||----------+\n";
```

Comment:
don't need parquet - keep it orc specific

Reply:
removing